scripts/standby_node_health_check/standby_node_health_check.py

```diff
@@ -0,0 +1,450 @@
+#
+# Copyright © 2013-2024 NVIDIA CORPORATION & AFFILIATES. ALL RIGHTS RESERVED.
+#
+# This software product is a proprietary product of Nvidia Corporation and its affiliates
+# (the "Company") and all right, title, and interest in and to the software
+# product, including all associated intellectual property rights, are and
+# shall remain exclusively with the Company.
+#
+# This software product is governed by the End User License Agreement
+# provided with the software product.
+#
+
+import json
+import re
+import subprocess
+import argparse
+import sys
+from typing import List
+
+
+def parse_arguments():
+    parser = argparse.ArgumentParser(description="Checking if standby node is ready.")
+
+    parser.add_argument(
+        "--fabric-interfaces",
+        nargs="+",
+        required=True,
+        help="Specify one or more fabric interfaces (at least one is required), eg ib0, mlx3_0",
+    )
+
+    parser.add_argument(
+        "--mgmt-interfaces",
+        nargs="+",
+        required=True,
+        help="Specify one or more management interfaces (at least one is required), eg eth0, eth1",
+    )
+
+    parser.add_argument("--version", action="version", version="%(prog)s 1.0.0")
+
+    args = parser.parse_args()
+    return args
+
+
+class StandbyNodeHealthChecker:
+    UFM_HA_CLUSTER_COMMAND = "ufm_ha_cluster {}"
+    SYSTEMCTL_IS_ACTIVE_COMMAND = "systemctl is-active {}.service"
+
+    def __init__(self, fabric_interfaces, mgmt_interfaces):
+        self._fabric_interfaces = fabric_interfaces
+        self._mgmt_interfaces = mgmt_interfaces
+        self._ufm_ha_status_string = ""
+
+    @classmethod
+    def _run_command(cls, command: str):
+        try:
+            result = subprocess.run(
+                command,
+                shell=True,
```

Comment:
I recommend to set "shell=False" by default, you can add it as an argument to the function.

Reply:
Changed to False, I don't think it should be an arg as it is also changes the command we need to pass to the run command (one string or a dict of string)